### PR TITLE
Tableau, SPSS Statistics, Alteryx, Matlab, Pentaho Data Integration updates

### PR DESCRIPTION
**Pull request by annstriganova**

### CODE REVIEW

File: integration/alteryx/README.md [hunk 1]
```diff
@@ -143,9 +143,9 @@ The workflow consists of the following steps (nodes):
    `inflation.cpi.categories.weight` table.
 
 3. **Filter** tool. Specify the condition `>= January 1st, 2010`
-   and use the **T** (_true_) node output to retrieve the series created after 2009 only.
+   and use the **T** (_true_) node output to retrieve the series created after 2010 only.
```
Comment (vtols) [inline integration/alteryx/README.md]: Series created in 2010 will be excluded?

File: integration/alteryx/README.md [hunk 2]
```diff
@@ -183,9 +183,8 @@ The workflow consists of the following steps (nodes):
 
    ![](images/sort.png)
 
-9. **Formula** tool. Add an `entity` column with the **Formula** tool. Name it
-   `bls.gov`. The default data type `V_WString` is not supported yet by the ATSD JDBC driver,
```
Comment (vtols) [inline integration/alteryx/README.md]: Does `V_WString` works now?

File: integration/alteryx/README.md [hunk 3]
```diff
@@ -154,9 +154,9 @@ The workflow consists of the following steps (nodes):
 5. **Join** tool. Join prices and weights by `tags.category` field. Deselect
    fields as shown on the image. Rename `value` fields for
    `inflation.cpi.categories.price` and `inflation.cpi.categories.weight` to
-   `price` and `weight` respectively.
+   `price` and `weight` respectively, `inflation.cpi.categories.price.datetime` to `datetime` and `inflation.cpi.categories.price.tags.category` to `tags.category`.
 
-   ![](images/join.png)
```
Comment (vtols) [inline integration/alteryx/README.md]: Delete join.png if you don't need it. Draw red rectangle as it was before for excluded fields - only in Input column

File: integration/alteryx/README.md (hunk 2 above)
Comment (annstriganova) [reply integration/alteryx/README.md]: Yes, it works.

File: integration/alteryx/README.md (hunk 1 above)
Comment (annstriganova) [reply integration/alteryx/README.md]: Thanks, you're right.

File: integration/alteryx/README.md (hunk 3 above)
Comment (annstriganova) [reply integration/alteryx/README.md]: Fixed.

**Pull request merged**